includes/Database_Upgrader.php

```diff
@@ -35,7 +36,7 @@
  *
  * @package Google\Web_Stories
  */
-class Database_Upgrader extends Service_Base implements Activateable {
+class Database_Upgrader extends Service_Base implements Activateable, Initialize_Site {
```

Comment:
Since this runs on `admin_init`, it won't necessarily be available during activation or initialization (think WP-CLI context).

We could change the registration action to `init` though and call `is_admin()` in `register()`.
`activate()` would then run the migration directly and not call `register()`.

Reply:
Again, this is what I get for developing something at 11pm without testing it.